theano/tensor/nnet/tests/test_sigm.py

```diff
@@ -237,27 +228,24 @@ def test_local_sigm_times_exp(self):
         def match(func, ops):
             # print [node.op.scalar_op for node in func.maker.fgraph.toposort()]
             assert [node.op for node in func.maker.fgraph.toposort()] == ops
+            assert check_stack_trace(f, ops_to_check=ops)
         m = self.get_mode(excluding=['local_elemwise_fusion', 'inplace'])
         x, y = tensor.vectors('x', 'y')
 
         f = theano.function([x], sigmoid(-x) * tensor.exp(x), mode=m)
-        assert hasattr(f.maker.fgraph.outputs[0].tag, 'trace')
         match(f, [sigmoid])
 
         f = theano.function([x], sigmoid(x) * tensor.exp(-x), mode=m)
-        assert hasattr(f.maker.fgraph.outputs[0].tag, 'trace')
         match(f, [tensor.neg, sigmoid])
```

Comment:
Actually, I think we should add a comment in [`local_sigm_times_exp`](https://github.com/lamblin/Theano/blob/fix_stack_check_nnet/theano/tensor/nnet/sigm.py#L911) or `perform_sigm_times_exp` to say we should add stack traces to the intermediate variables at some point, and open a low-priority ticket about it.